src/Middlewares/isPostCreator.ts

```diff
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from 'express';
+import RequestError from '../Classes/Errors/RequestError';
+import PostDBConnector from '../Classes/PostDBConnector/PostMongoDBConnector';
+import getUser from '../Helpers/getUserFromQuery';
+import Admin from '../Models/Admin';
+import Post from '../Models/Post';
+
+const isPostCreator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const postDBConnector = new PostDBConnector();
+    const { id } = req.params;
+    const post = (await Post.getById(postDBConnector, id)) as Post;
+    const user = await getUser(req);
+    if (!(user instanceof Admin) || post.creator === user.id) {
+      const error = new RequestError('Authorization Problem', 403);
```

Comment:
please extract message text (Authorization Problem) and status code to some file with constants. PLease apply this to all error messages. Please note, naming for constants in this case should be CAPITALIZED_AND_UNDERSCORED